internal/bgp/native/native.go

```diff
@@ -665,20 +658,20 @@ func dialMD5(ctx context.Context, addr string, srcAddr net.IP, password string)
 	}
 }
 
-func buildTCPMD5Sig(addr net.IP, key string) tcpmd5sig {
```

Comment:
we also have the struct definition we can get rid of, I guess

Reply:
Thanks, removed the struct definition as well.